content/spin/spin-oci.md

```diff
@@ -0,0 +1,94 @@
+title = "Hosting Spin Apps in OCI registries"
```

Comment:
Not sure if "hosting" is the right verb here, and we have been talking a bit in the Spin project about not using "OCI" to talk about the registries necessarily.

"Distributing Spin apps using container registries" might be a better title here?